### ISSUE TITLE
goes forever?

### ISSUE THREAD

**Issue by elazzzzyy** [liamg/scout#21]
This has been fuzzing a url for about an hour and 46 mins but also shows me all outputs like 404, 503, 200 despite some code 200's being blank pages.

**Comment by Routhinator** [comment]
Yeah, I'm having a similar problem with a page that uses an angular client. Since angular will show the 404 page from the client on every request, it returns a 200, and since this doesn't check to see if the contents of the response are actually a valid find and not the same default page, it's of little value when pointed at a frontend client that doesn't have server side rendering (like Angular Universal)

Works ok when I point it at the /api/ url though.
